Mathlib/RingTheory/IsTensorProduct.lean

```diff
@@ -382,6 +382,7 @@ class Algebra.IsPushout : Prop where
 
 variable {R S R' S'}
 
+@[symm]
 theorem Algebra.IsPushout.symm (h : Algebra.IsPushout R S R' S') : Algebra.IsPushout R R' S S' := by
```

Comment:
Does `symm` really work on this one?

Reply:
Yes, I checked the ones that I was unsure about, it seems symm is more versatile than I understood it to be